### PR TITLE
Hw 5

### PR DESCRIPTION
**Pull request by Giliaz**

### CODE REVIEW

File: hw_5/point_hw5.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class Circle:
+    def __init__(self, x=0, y=0, radius=10):
+        self.x = x
+        self.y = y
+        self.radius = radius
+        print(self.x, self.y, self.radius)
+
+    def in_circle(self, f):
```
Comment (mikolaz27) [inline hw_5/point_hw5.py]: in requirements, we have method contains.

File: hw_5/point_hw5.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        print(f.x,  f.y)
+        return (self.x - f.get_x()) ** 2 + (self.y - f.get_y()) ** 2 < self.radius ** 2
```
Comment (mikolaz27) [inline hw_5/point_hw5.py]: you can use just atr of f class, like f.x or f.y

File: hw_5/point_hw5.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+class Circle:
+    def __init__(self, x=0, y=0, radius=10):
+        self.x = x
+        self.y = y
+        self.radius = radius
+        print(self.x, self.y, self.radius)
+
+    def in_circle(self, f):
+        print(f.x,  f.y)
```
Comment (mikolaz27) [inline hw_5/point_hw5.py]: remove prints

File: hw_5/point_hw5.py (hunk 1 above)
Comment (mikolaz27) [inline hw_5/point_hw5.py]: rename f variable with some meaningful name, like point, and its a good idea to add an annotation.

File: hw_5/point_hw5.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        self.x = x
+        self.y = y
+
+    def get_x(self):
+        return self.x
+
+    def get_y(self):
+        return self.y
+
+#circle x, y, radius
```
Comment (mikolaz27) [inline hw_5/point_hw5.py]: remove comments

File: hw_5/point_hw5.py (hunk 1 above)
Comment (Giliaz) [reply hw_5/point_hw5.py]: fix

File: hw_5/point_hw5.py (hunk 2 above)
Comment (Giliaz) [reply hw_5/point_hw5.py]: fix

File: hw_5/point_hw5.py (hunk 3 above)
Comment (Giliaz) [reply hw_5/point_hw5.py]: fix

File: hw_5/point_hw5.py (hunk 1 above)
Comment (Giliaz) [reply hw_5/point_hw5.py]: fix

File: hw_5/point_hw5.py (hunk 4 above)
Comment (Giliaz) [reply hw_5/point_hw5.py]: fix

**Review by mikolaz27 — APPROVED** [review]

**Pull request merged**